lookup_plugins/json_template.py

```diff
@@ -0,0 +1,65 @@
+# (c) 2018 Red Hat, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+lookup: json_template
+author: Ansible Network
+version_added: "2.5"
+short_description: retrieve and template device configuration
+description:
+  - This lookup plugin returns the content of a JSON file in JSON format.
+    configuration.
+options:
+  _terms:
+    description: list of files for lookup
+"""
+
+EXAMPLES = """
+- name: show interface lookup result
+  debug: msg="{{ lookup('json_template', './show_interface.json') }}
+"""
+
+RETURN = """
+_raw:
+   description: JSON file(s) content
+"""
+
+import json
+
+from ansible.errors import AnsibleError, AnsibleParserError
+from ansible.plugins.lookup import LookupBase
+from ansible.module_utils._text import to_bytes
+from ansible.module_utils.network.common.utils import to_list
+
+try:
+    from __main__ import display
+except ImportError:
+    from ansible.utils.display import Display
+    display = Display()
+
+
+class LookupModule(LookupBase):
+
+    def run(self, terms, variables, **kwargs):
+        ret = list()
+
+        for term in to_list(terms[0]):
```

Comment:
This should only use a single path not a list of paths.  Change `to_list(terms[0])` to `terms[0]` and remove the `for` loop